internal/grpc/services/gateway/webdavstorageprovider.go

```diff
@@ -238,6 +238,7 @@ func (s *svc) extractEndpointInfo(ctx context.Context, targetURL string) (*webda
 }
 
 func (s *svc) getWebdavEndpoint(ctx context.Context, domain string) (string, error) {
+	fmt.Printf("getWebdavEndpoint( domain = %v )\n", domain)
```

Comment:
please make sure to use the logger with an appropriate log level or remove this (development/debug?) statement